source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -391,6 +391,15 @@ namespace AdaptiveSharedNamespace
     };
     // No mapping to string needed
 
+    enum class ContainerBleedState
+    {
+        BleedRestricted = 0,
+        BleedToLeading,
+        BleedToTrailing,
+        BleedToBothEdges,
+    };
```

Comment:
Think this might read a bit better like this:

```
    enum class ContainerBleedType
    {
        None = 0,
        ToLeadingEdge,
        ToTrailingEdge,
        ToBothEdges,
    };
```

Reply:
thanks; will do

---
In reply to: [265290586](https://github.com/Microsoft/AdaptiveCards/pull/2544#discussion_r265290586) [](ancestors = 265290586)